sources/packages/web/src/composables/useFormioUtils.ts

```diff
@@ -117,6 +117,33 @@ export function useFormioUtils() {
     return matchedComponents;
   };
 
+  /**
+   * Search recursively by a component and returns
+   * all the matches with the same key.
+   * @param components components to be checked.
+   * @param componentKey key to be matched.
+   * @param options search options.
+   * - `stopOnFirstMatch` indicates if should stop at the first match, default true.
+   * @returns components found.
+   */
+  const searchByKey = (
+    components: FormIOComponent[],
+    componentKey: string,
+    options?: {
+      stopOnFirstMatch: boolean;
+    },
+  ): FormIOComponent[] => {
+    const defaultOptions = { stopOnFirstMatch: true };
```

Comment:
Why is `stopOnFirstMatch` not used here?

Reply:
`stopOnFirstMatch` is used as `true` and is the default option. I am not following the question. Can you please clarify?

Reply:
I missed the default part. resolving.